src/testing/loadtest-azure/infra/main.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     azurerm = {
       source  = "hashicorp/azurerm"
-      version = "3.4.0"
+      version = "3.5.0"
```

Comment:
Sneaky terraform update in chaos optimizations? ;)

Reply:
I wonder why this wasn't updated by dependabot in yesterday's PR #330

Reply:
it wasnt in the dependabot file then. I just updated that today

Reply:
Ah - it's not dependabot's fault. Someone closed PR #390 ;)

Reply:
haha yeah since this PR will anyway bring it to main directly